### PR TITLE
added trial/block counter

### PR DESCRIPTION
**Pull request by svr8**
This is a fix for #8

### CODE REVIEW

File: threshold.js [hunk 1]
```diff
@@ -15,6 +15,7 @@ import * as jsQUEST from "./lib/jsQUEST.module.js";
 window.jsQUEST = jsQUEST;
 
 var conditionTrials;
+var blocksPerTrial;
```
Comment (peilingjiang) [inline threshold.js:18]: What do you mean by blocks per trial? A block contains multiple trials.

File: threshold.js [hunk 2]
```diff
@@ -304,6 +317,20 @@ const experiment = (blockCount) => {
       opacity: 1.0,
       depth: -7.0,
     });
+    stepCount = new visual.TextStim({
```
Comment (peilingjiang) [inline threshold.js:320]: Structure-wise I don't suggest breaking code of flankers and target apart and they are one unit.

File: threshold.js (hunk 1 above)
Comment (svr8) [reply threshold.js:18]: Okay, I thought it was the otherway. Then it should have been trialPerBlocks?

**Comment by peilingjiang** [comment]
@svr8 Looks good! I rebase the current main and fixed some problems. Two things that are not working:
- The trial starts from 0 instead of 1, can you check that?
- Don't use the absolute placement, you can refer to the position of valid answers (showAlphabet) to see how to use PsychoJS's relative positioning to make sure the text is always at the corner for all screen sizes.

**Comment by svr8** [comment]
> * Don't use the absolute placement, you can refer to the position of valid answers (showAlphabet) to see how to use PsychoJS's relative positioning to make sure the text is always at the corner for all screen sizes.

I've used alignment properties and then used positioning relative to window now. Is this right?

**Review by peilingjiang — APPROVED** [review]

**Pull request merged**